### ISSUE TITLE
Bump github.com/google/cadvisor from 0.47.3 to 0.48.1 broke /receiver/awscontainerinsightreceiver

### ISSUE THREAD

**Issue by songy23** [open-telemetry/opentelemetry-collector-contrib#28995]
### Component(s)

receiver/awscontainerinsight

### What happened?

See https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/28974

```
Error: ../../receiver/awscontainerinsightreceiver/internal/cadvisor/cadvisor_linux.go:55:77: undefined: manager.HouskeepingConfig
Error: ../../receiver/awscontainerinsightreceiver/internal/cadvisor/cadvisor_linux.go:64:113: undefined: manager.HouskeepingConfig
Error: ../../receiver/awscontainerinsightreceiver/internal/cadvisor/cadvisor_linux.go:377:32: undefined: manager.HouskeepingConfig
```

### Collector version

mainline

**Comment by crobert-1** [comment]
The variable name was originally a typo, fixed in v0.48.1.

Old name:
```
manager.HouskeepingConfig
```
New name:
```
manager.HousekeepingConfig
``` 

[Type that needs referenced](https://github.com/google/cadvisor/blob/2129e1c4fae8567551d1ac252c7a1dec27617e21/manager/manager.go#L149)
[Fix commit](https://github.com/google/cadvisor/commit/4dcb908c5728c9313dc74d5805c44665f65f2193)

**Comment by crobert-1** [comment]
I'll open a PR shortly with the fix
